### DISCUSSION TITLE
Reprogram column headers to the old configuration

### DISCUSSION THREAD

**Discussion by cwpeters** [community/community#21430] [Other Feature Feedback, Questions, & Ideas]
<p>Hello<br>
I am Calvin<br>
I am trying to find a way to change column headers to what i want., Other than what is by default always there.  " Subject, correspondence  and date".  The way ti used to be way back when.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>That sounds like you’re having trouble with a specific piece of software, nothing related to Git or GitHub. If that software is hosted on GitHub, it’s usually best to open an issue in the repository. If there’s a README check that first for info on how to get support, though. Good look! :slightly_smiling_face:</p>
